File: SAMPLES/JAVA/test08_string.java
public class test08_string {
    public static void main(String[] args) {
        String s;
        String t;
        String u;

        s = "Bonjour";
        t = " monde";
        u = s + t;
        System.out.println(u);

        System.out.println("Longueur : " + s);

        s = "Java" + " " + "Lite";
        System.out.println(s);

        System.out.println("Entrez un texte :");
        t = Scanner.nextLine();
        System.out.println(t);
    }
}

File: SAMPLES/CSHARP/test08_string.cs
using System;

namespace Test08
{
    class Program
    {
        static void Main()
        {
            string s1;
            string s2;
            string s3;

            s1 = "Bonjour";
            s2 = " monde";
            s3 = s1 + s2;
            Console.WriteLine(s3);

            s3 = "Valeur : " + "42";
            Console.WriteLine(s3);

            Console.WriteLine("Entrez votre nom :");
            s1 = Console.ReadLine();
            s2 = "Salut, " + s1;
            Console.WriteLine(s2);
        }
    }
}
